index.html

```diff
@@ -67,227 +78,271 @@ <h2 class="my-position">I’am a Software Developer</h2>
+                        <li class="language-list-content">
+                            CapyBara
+                        </li>
+                        <li class="language-list-content">
+                            Selenium
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <div class="skill">
+                <div class="language skills">
+                    <img alt="Vector" class="orange-vector" src="css/Icons/orange_vectors/circle.png">
+                    <h2 class="lang-h2">Skills</h2>
+                </div>
+                <div class="language-names">
+                    <ul class="language-list">
+                        <li class="language-list-content">
+                            Codekit
+                        </li>
+                        <li class="language-list-content">
+                            Github
+                        </li>
+                        <li class="language-list-content">
+                            Codepen
+                        </li>
+                        <li class="language-list-content">
+                            Gitlab
+                        </li>
+                        <li class="language-list-content">
+                            Terminal
+                        </li>
+                    </ul>
+                </div>
             </div>
         </div>
     </section>
     <section class="lastpage">
-        <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
-            please get in touch.</h2>
-        <form class="form" action="https://formspree.io/f/xnqwwrzy" method="POST">
-            <input class="fullname" type="text" name="fullname" placeholder="Full name" required>
-            <input class="fullname" type="email" name="fullname" placeholder="Email address" required>
-            <textarea class="text-area" placeholder="Enter text here" required></textarea>
-            <button type="submit" class="button1 button1-text button-end">Get in toucht</button>
-        </form>
+        <div class="contact-form">
+            <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
+                please get in touch.</h2>
+            <form class="form" action="https://formspree.io/f/xnqwwrzy" method="POST">
+                <input class="fullname" type="text" name="fullname" placeholder="Full name" required>
+                <input class="fullname" type="email" name="fullname" placeholder="Email address" required>
+                <textarea class="text-area" placeholder="Enter text here" required></textarea>
+                <button type="submit" class="animate-btn button1 button1-text button-end btn-d">Get in touch</button>
+            </form>
+        </div>
         <hr class="footer-line1">
         <ul class="icon-list last-page-footer">
             <li class="img-list">
```

Comment:
- [ **OPTIONAL** ] Good job so far by making all changes I required before. So, I also think it would be better to make the ` <hr class="footer-line1">` horizontal line to have a `100%` of the page and improve the **height** of the footer to stick to the one of the Figma template, you can check on the `margins` and `paddings` you used on its content. But great job :+1: :+1: